tests/modules/services/ollama/load-models.nix

```diff
@@ -0,0 +1,17 @@
+{
+  config = {
+    services.ollama = {
+      enable = true;
+      loadModels = [ "llama2" ];
+    };
+
+    test.stubs.ollama = { };
+
+    nmt.script = ''
+      serviceFile="home-files/.config/systemd/user/ollama.service"
+      assertFileRegex "$serviceFile" 'ExecStartPost=/nix/store/.*-ollama-post-start'
+      generated="$(grep -o '/nix/store/.*-ollama-post-start' "$TESTED/home-files/.config/systemd/user/ollama.service")"
+      assertFileContains "$generated" "for model in 'llama2'"
```

Comment:
can this test run without network, i suspect it fails due to not being able to download the model in time

Reply:
I don't think the test will execute the service, but we just check the content of the generated files.

Reply:
i misunderstood the intent of the test here, but im now absolutely puzzled as to why it fails on ci

Reply:
Okay, a typo, or something changed. It is generated as `for model in llama2` and not `for model in 'llama2'`